### PR TITLE
multi: Allow empty trezor passhrases.

### LINKED ISSUE #3460: [trezor] support empty passphrase on passphrase-enabled wallets
The passphrase modal currently requires a non-empty passphrase, but when using the trezor passphrase feature, you might be using a wallet with an empty passphrase (the "default" trezor wallet).

### PR DESCRIPTION
**Pull request by JoeGruffins**
In order to use the default account with passphrase enabled on trezor,
    allow empty passphrases.

    Show a confirmation message upon toggling passphrase protection. Inform
    the user that they should unplug and plug back in their device in order
    for changes to take effect.

closes #3460

### CODE REVIEW

File: app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const enable = enablePassphraseProtection ? "Enable" : "Disable";
+  const enabled = enablePassphraseProtection ? "enabled" : "disabled";
+  const ifEnabled = enablePassphraseProtection
+    ? " For your first sensitive operation, " +
+      "the device will ask for a passphrase. This pasphrase deterministically creates a " +
+      "new seed for your Trezor. If used with an already created wallet, subsequent " +
+      "actions will likely fail, as the device is now using a different account/wallet. " +
+      "As long as the device stays plugged in, further actions will reuse the first " +
+      "passed passphrase and not ask again."
+    : "";
```
Comment (matheusd) [inline app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx:21]: These need to be translatable.

**Review by matheusd — COMMENTED** [review]
Tested fine for supplying the empty passphrase when sending txs, but wallet restore is still requiring a non-empty passphrase.

File: app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx (hunk 1 above)
Comment (JoeGruffins) [reply app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx:21]: yeah, not even sure how to do that, will figure it out though

Comment (JoeGruffins) [reply app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx]: Think this got it.

File: app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx [hunk 2]
```diff
@@ -9,16 +9,31 @@ const TrezorTogglePassphraseConfirmModal = ({
...
+  const enabled = enablePassphraseProtection ? (
+    <T id="trezor.togglePassphraseConfirmModal.enabled" m="enabled" />
+  ) : (
+    <T id="trezor.togglePassphraseConfirmModal.disabled" m="disabled" />
+  );
+  const ifEnabled = enablePassphraseProtection ? (
+    <T
+      id="trezor.togglePassphraseConfirmModal.ifEnabled"
+      m={
+        " For your first sensitive operation, " +
```
Comment (matheusd) [inline app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx:26]: Extra space at the start

Comment (JoeGruffins) [reply app/components/modals/TrezorModals/TrezorTogglePassphraseConfirmModal.jsx]: This better?

**Review by matheusd — APPROVED** [review]
Tested restoring and using wallets with and without passphrase

**Pull request merged**